CollAction/Services/ProjectService.cs

```diff
@@ -26,6 +27,11 @@ public ProjectService(ApplicationDbContext context, IStringLocalizer<IProjectSer
             return await _context.Projects.SingleOrDefaultAsync(p => p.Id == id);
         }
 
+        public async Task<Project> GetProjectByName(string name)
+        {
+            return await _context.Projects.SingleOrDefaultAsync(p => p.Name == name);
```

Comment:
We need a unique index on the project name. All these queries on the name aren't going to scale if collaction ever gets big.

Reply:
Next time you can add a new commit to the pull request, no need to make a new one.

Reply:
It has been decided that due to scope of changes a commit squash is needed.